app/src/main/java/com/quran/labs/androidquran/ui/PagerActivity.java

```diff
@@ -185,9 +185,9 @@
   private ViewGroup.MarginLayoutParams audioBarParams;
   private boolean isInMultiWindowMode;
 
-  private String[] translationItems;
-  private List<LocalTranslation> translations;
-  private Set<String> activeTranslations;
+  private String[] translationSpinnerTitles;
+  private List<LocalTranslation> allTranslationsFromDb;
```

Comment:
maybe `translations` is better since if we decide to read it from somewhere else (preferences, etc, etc) this variable won't be correctly named.

Reply:
ok